modules/ROOT/pages/salesforce/salesforce-connector-reference-96.adoc

```diff
@@ -1374,7 +1381,7 @@ Creates a Bulk API v2 job containing the data needs to be inserted, updated, del
 ** JSON
 ** ZIP_XML
 ** ZIP_CSV
-** ZIP_JSON |  |  | 
+** ZIP_JSON |The only value allowed is ZIP_CSV. If any other value is selected, an exception is thrown. The error is an INVALID REQUEST DATA with the message `The only content type supported is CSV`. | ZIP_CSV | 
```

Comment:
The only value allowed is CSV not ZIP_CSV.

Reply:
Fixed as requested.